golog.go

```diff
@@ -91,7 +91,12 @@ func GetPrepender() func(io.Writer) {
 
 // SetOutputs sets the outputs for error and debug logs to use the given Outputs.
 // Returns a function that resets outputs to their original values prior to calling SetOutputs.
+// If env variable PRINT_JSON is set, use JSON output instead of plain text
 func SetOutputs(errorOut io.Writer, debugOut io.Writer) (reset func()) {
+	if printJson, _ := strconv.ParseBool(os.Getenv("PRINT_JSON")); printJson {
```

Comment:
I think this would be better to do in ResetOutputs(), because I _think_ what you're saying with the PRINT_JSON is that you want to default to printing JSON if no one has called SetOutputs yet, right?

Reply:
@oxtoacart - not really. SetOutputs seems to be used by ResetOutputs only, and I'm not enabling JSON by default, only if the right ENV variable is set.
So for all current users of golog - nothing changes. But once you set PRINT_JSON=true before running flashlight (after it's updated to new glog) - it will start spitting out JSON lines

Reply:
Ah good point, this is just taking writers, not an Output. I'm cool with it then.